packages/fxa-settings/src/pages/InlineTotpSetup/index.tsx

```diff
@@ -0,0 +1,320 @@
+                      <p className="text-sm m-4">
+                        Scan the QR code in your authentication app and then
+                        enter the security code it provides.{' '}
+                        <button
+                          type="button"
+                          onClick={() => {
+                            setShowQR(false);
+                          }}
+                          className="link-blue inline"
+                        >
+                          Can’t scan code?
+                        </button>
+                      </p>
+                    </FtlMsg>
+                    <div>
+                      <img
+                        className={classNames({
+                          hidden: !qrCodeSrc,
+                        })}
+                        alt={localizedQRCodeAltText}
+                        src={qrCodeSrc}
+                      />
+                    </div>
+                    <FtlMsg id="inline-totp-setup-on-completion-description">
+                      <p className="text-sm m-4">
+                        Once complete, it will begin generating security codes
+                        for you to enter.
+                      </p>
+                    </FtlMsg>
+                  </>
+                ) : (
+                  <>
+                    <FtlMsg
+                      id="inline-totp-setup-enter-key-or-use-qr-instructions"
+                      elems={{
+                        toggleToQRButton: (
+                          <button
+                            onClick={() => {
+                              setShowQR(true);
+                            }}
+                            className="link-blue inline"
+                          >
+                            Scan QR code instead?
+                          </button>
+                        ),
+                      }}
+                    >
+                      <p className="text-sm m-2">
+                        Type this secret key into your authentication app.{' '}
+                        <button
+                          onClick={() => {
+                            setShowQR(true);
+                          }}
+                          className="link-blue inline"
+                        >
+                          Scan QR code instead?
+                        </button>
+                      </p>
+                    </FtlMsg>
+                    <div className="qr-code-container">
+                      <div className="qr-code-text">{secret}</div>
+                    </div>
+                    <FtlMsg id="inline-totp-setup-on-completion-description">
+                      <p className="text-sm m-4">
+                        Once complete, it will begin generating security codes
+                        for you to enter.
+                      </p>
+                    </FtlMsg>
+                  </>
+                )}
+                <form
+                  noValidate
```

Comment:
You could use the new `FormVerifyCode` component for this section (it's now landed in main!) 😄